src/uu/mkdir/mkdir.md

```diff
@@ -0,0 +1,11 @@
+# mkdir
+
+```
+mkdir [OPTION]... [USER]
+```
+
+Create the given DIRECTORY(ies) if they do not exist
+
+## Long Usage
```

Comment:
Please call this "After help" for consistency with other utils. You can also change the name of the variable in the code.

Reply:
Thanks! I have changed the naming as you suggested.